### PR TITLE
add dbt power user link

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds links the dbt power user link in the cloud cli doc for add'l context.

### CODE REVIEW

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 1]
```diff
@@ -251,10 +251,14 @@ To update:
 
 ## Using VS Code extensions
 
-Visual Studio (VS) Code extensions enhance command line tools by adding extra functionalities. The dbt Cloud CLI is fully compatible with dbt Core, however it doesn't support some dbt Core APIs required by certain tools, for example VS Code extensions. 
+Visual Studio (VS) Code extensions enhance command line tools by adding extra functionalities. The dbt Cloud CLI is fully compatible with dbt Core, however, it doesn't support some dbt Core APIs required by certain tools, for example, VS Code extensions. 
 
-To use these extensions, such as dbt-power-user, with the dbt Cloud CLI, you can install it using Homebrew (along with dbt Core) and create an alias to run the dbt Cloud CLI as `dbt-cloud`. This allows dbt-power-user to continue to invoke dbt Core under the hood, alongside the dbt Cloud CLI.
+You can use extensions like [dbt-power-user](https://www.dbt-power-user.com/) with the dbt Cloud CLI by following these steps:
 
+- Install it using Homebrew along with dbt Core.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]: These aren't really "steps" as there are no instructions. Should we show how to install from homebrew and create an alias in this section or at least link to the instructions?

Comment (mirnawong1) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
- [Install](/docs/cloud/cloud-cli-installation?install=brew) it using Homebrew along with dbt Core.
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 2]
```diff
@@ -251,10 +251,14 @@ To update:
...
+- Create an alias to run the dbt Cloud CLI as `dbt-cloud`.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
- [Create an alias](#faqs) to run the dbt Cloud CLI as `dbt-cloud`.
```

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/cloud-cli-installation.md]: the homebrew installation is at the beginning of the page so i don't think we should duplicate instructions due to maintenance but happy to link to it!

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**